pkg/controller/storagecluster/storageclasses.go

```diff
@@ -14,7 +14,7 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
-const(
+const (
```

Comment:
This change is unrelated and being taken care of in another PR, please remove it.